benchmark/pom.xml

```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>io.prometheus</groupId>
             <artifactId>simpleclient</artifactId>
-            <version>0.10.1-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
This will break some build systems, I forget which one but that's the reason all versions are explicit.

Reply:
reverted